config.md

```diff
@@ -90,7 +90,9 @@ See links for details about [mountvol](http://ss64.com/nt/mountvol.html) and [Se
 
 ## Process configuration
 
-* **`terminal`** (bool, optional) specifies whether you want a terminal attached to that process, defaults to false.
+* **`terminal`** (bool, optional) specifies whether a terminal is attached to that process, defaults to false.
+  On Linux, a psuedoterminal pair is allocated for the container process and the psuedoterminal slave is duplicated on the container process's [standard streams][stdin.3].
+  Also, the psuedoterminal slave is bind mounted at /dev/console.
```

Comment:
Actually, /dev/console looks like a different beast from the psuedoterminal.  Why link them at all?